Remove compiler option which requires a gcc newer than 4.0.1 and which,
according to the author, is not important (suppresses a harmless warning).
--- configure.orig	2009-04-06 09:50:38.000000000 -0500
+++ configure	2009-04-25 16:13:45.000000000 -0500
@@ -3458,7 +3458,7 @@
 
 
 if test "$GCC" = "yes" ; then
-	CFLAGS="$CFLAGS -Wall -pedantic -Wstrict-overflow=0"
+	CFLAGS="$CFLAGS -Wall -pedantic"
 fi
 
 # Checks for libraries.
